source/switching_to_live/set_up_a_live_stripe_account/index.html.md.erb

```diff
@@ -0,0 +1,25 @@
+---
+title: Connect your live account to Stripe
+weight: 100
+---
+
+# Connect your live account to Stripe
+
+First make sure you've:
+
+- read about [what to do first](/switching_to_live/#before-you-start)
+- [requested a live GOV.UK Pay account](/switching_to_live/#1-request-a-live-gov-uk-pay-account)
+
+After you request a live GOV.UK Pay account, we'll contact you to verify your organisation’s identity.
+
+We'll also give you a link where you can provide your organisation's:
```

Comment:
So... Services can take payments as soon as we have made their account live, before they have submitted the additional information. If a service doesn't submit the information in time, their account is disabled. 

We communicate that to them when we email them that their service is live, but we've had issues with services not completing the information in time. Is it worth including that here?

Reply:
This does sound worth including, but probably only if we can give a specific deadline in days/weeks/etc. Do we have one?

Reply:
I don't think we're quite clear on that, but I can try to confirm with Stripe.

Reply:
That'd be good! To avoid holding up this PR, I think we're probably ok to leave as 1 day for now, as that's what we've had for a while?

Reply:
Tweaked in https://github.com/alphagov/pay-tech-docs/pull/356/commits/e9bf6d29ab93cfba4848afccd8605cefe780eeff as discussed!

Reply:
Sounds good!